src/test/resources/com/jpinpoint/perf/lang/java/ruleset/common/xml/ImproperVariableName.xml

```diff
@@ -11,14 +11,38 @@
 class Foo {
     private static final int FOUR_ZERO_NINE_SIX = 4096; // bad
     private static int six = 6; // bad
-    private int five = 6; // really bad
+    private int five = 6; // fire this guy ;-)
     private static final int SIXTIES_START = 1960; // good
     long three; // bad - missing case #240
     String tree; //good - false positive #240
+    String HIGH_FIVE; // good
+
+    int VAL_127 = 127; //bad #323
+    String VALUE_127 = "127"; //bad #323
+    int VAL_ONE_TWO_SEVEN; // bad #323
+    int INT_5 = 5; //bad #323
+    int INTEGER_5 = 5; //bad #323
+    long long_7 = 7; //bad #323
+    int NUMBER_7 = 7; //bad #323
+    int NUM_7 = 7; // bad #323
+    int TWENTY_EIGHT; // bad #323 - line 18
+
+    int MAX_INT; // good
+    int MIN_VAL = 3; // good
+    int IN_5 = 5; // good
+    int INTEG_FIVE; // good
+    int MAMBO_NUMBER_FIVE; // good
+
+    static final int VALUE_5 = 5; // bad - line 26
 
     void bar() {
-        String var1 = "baz"; // bad
+        String var1 = "baz"; // bad - line 29
```

Comment:
"line 29" is for the expected-linenumbers? are these also updated?

Reply:
java test file was niet verplaatst en unit test liep daardoor niet. Regelnummers ook aangepast, 15 nu.